protobufs/livekit_rtc.proto

```diff
@@ -160,6 +172,43 @@ message AddTrackRequest {
   repeated AudioTrackFeature audio_features = 17;
 }
 
+message PublishDataTrackRequest {
+  // 16-bit identifier attached to packets, unique per publisher.
+  uint32 handle = 1;
+
+  // Human-readable identifier (e.g., `geoLocation`, `servoPosition.x`, etc.), unique per publisher.
+  string name = 2;
+
+  // MIME type of the data sent over the track (e.g., `application/json`).
+  // This must be a valid MIME type as defined by RFC 2046.
+  string mime_type = 3;
+
+  // Method used for end-to-end encryption (E2EE) on frame payloads.
+  Encryption.Type encryption = 4;
+}
+
+message PublishDataTrackResponse {
+  // Handle of the track that was published.
+  uint32 handle = 1;
```

Comment:
If `handle` in included in the model, this field can be dropped from this message.